app/src/main/java/openfoodfacts/github/scrachx/openfood/fragments/NutritionInfoProductFragment.java

```diff
@@ -97,7 +97,8 @@ public void onViewCreated(View view, @Nullable Bundle savedInstanceState) {
         List<NutrimentItem> nutrimentItems = new ArrayList<>();
 
         if (isNotBlank(product.getServingSize())) {
-            mTextPerPortion.setText(getString(R.string.nutriment_serving_size) + " " + product.getServingSize());
+//            barcodeText.setText(getString(R.string.format_barcode_text, barcodeText.getText(),  mBarcode));
+            mTextPerPortion.setText(getString(R.string.nutriment_serving_size, Integer.getInteger(product.getServingSize())));
```

Comment:
Do you want to convert the String value to an Integer value ? Because `Integer.parseInt / Integer.valueOf` is the function to use for that.